operations/workplace/people/working-at-mattermost/paid-time-off/README.md

```diff
@@ -2,43 +2,45 @@
+Mattermost is a high-performance, fast-paced startup where staff are empowered to do the best work of their careers. We love what we do and we love doing it from anywhere, working remotely and often blending our work and personal lives.
 
-In many ways, Mattermost can be "the Olympics" of our professional careers, with the opportunity to grow and achieve far beyond anywhere we've worked before.
+In many ways, Mattermost can be "the Olympics" of our professional careers, with the opportunity to grow and achieve far beyond anywhere we've worked before. At the same time, like star athletes, our growth and peak performance benefit greatly from the ability to interleave periods of exertion with periods of rest and reflection.
 
-At the same time, like star athletes, our growth and peak performance benefit greatly from the ability to interleave periods of exertion with periods of rest and reflection.
-
-Paid Time Off \("[PTO](../../../../../company/about-mattermost/list-of-terms.md#pto-or-paid-time-off)"\) is a process to help ensure our staff develop a strong practice of **self-care** by effectively rotating their time around work, family, friends and all the things that matter most to them.
+Planned Time Off \("[PTO](../../../../../company/about-mattermost/list-of-terms.md#pto-or-paid-time-off)"\) is a process to help ensure our staff develop a strong practice of **self-care** by effectively rotating their time around work, family, friends and all the things that matter most to them. Mattermost is committed to supporting the well-being and work-life balance of our staff through our Unlimited PTO policy. This policy allows our team members the flexibility to take time off as needed, encouraging rest and making sure everyone is bringing their best selves to work.
 
 Note: If you are taking [Pregnancy](../leaves-of-absence/pregnancy-leave.md) or [Parental Leave](../leaves-of-absence/pregnancy-leave.md), please see [Leave Policies](../leaves-of-absence/).
 
+## Statutory Vacation Requirements
+There are statutory vacation entitlements in countries where Mattermost has an entity or EOR. At a minimum, staff must use their statutory vacation entitlement each year and it is the responsibility of managers to ensure that their direct reports are taking the required statutory minimum vacation. We understand there may be circumstances where, due to exceptional operational needs or personal issues, it isn’t possible to use up the full annual statutory vacation entitlement.  Exceptions may be granted for vacation carry over in those situations or where there is a statutory requirement to do so. Statutory vacation day carryover must be approved in advance by your manager, Team MLT, and the People Team. Staff members should work with their managers to discuss a plan for booking any accrued but unused statutory vacation. 
```

Comment:
There may be statutory vacation entitlements in countries where Mattermost has an entity or EOR.